### PR TITLE
Fixed formatting. Fixed broken image links. Fixed section links.

### LINKED ISSUE #186: Broken image links and page links in documentation files.
- [ ] Broken image links in Device Twin docs  

- [ ] Broken page links in What is KubeEdge page.  

- [ ] Long link titles in Sidebar menu

- [ ] Markdown tables not parsed in html files.

### PR DESCRIPTION
**Pull request by anvithks**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

- [x] The images were not visible in the Device Twin page. 

- [x] The links to pages from the What is KubeEdge page were broken. 

- [x] The link titles in the sidebar were very long. 

- [x] The markdown tables were not being parsed.

This PR fixes all of these issues.
**Which issue(s) this PR fixes**:
Fixes #186 

**Special notes for your reviewer**:
This is the working [preview](https://kubeedge-anvith.readthedocs.io/en/update-links/index.html)
The tables were incorrectly displayed as [here](http://docs.kubeedge.io/en/latest/modules/devicetwin.html#device-table).  
They have been fixed as [here](https://kubeedge-anvith.readthedocs.io/en/update-links/modules/devicetwin.html#tables)

### CODE REVIEW

**Comment by anvithks** [comment]
@rohitsardesai83 @m1093782566  Please review and merge.

File: docs/kubeedge.md [hunk 1]
```diff
@@ -26,10 +26,10 @@ The advantages of Kubeedge include mainly:
...
-- **[DeviceTwin](modules/devicetwin):** DeviceTwin is responsible for storing device status and syncing device status to the cloud. It also provides query interfaces for applications.
-- **[MetaManager](modules/metamanager):** MetaManager is the message processor and between edged and edgehub. It's also responsible for storing/retrieving metadata to/from a lightweight database(SQLite). 
+- **[EdgeHub](modules/edgehub.html):** EdgeHub is a web socket client, which is responsible for interacting with **Huawei Cloud IEF service**, including sync cloud side resources update, report edged side host and device status changes.
+- **[EventBus](modules/eventbus.html):** EventBus is a MQTT client to interact with MQTT server(mosquitto), offer subscribe and publish capability to other components.
+- **[DeviceTwin](modules/devicetwin.html):** DeviceTwin is responsible for storing device status and syncing device status to the cloud. It also provides query interfaces for applications.
+- **[MetaManager](modules/metamanager.html):** MetaManager is the message processor and between edged and edgehub. It's also responsible for storing/retrieving metadata to/from a lightweight database(SQLite).
```
Comment (rohitsardesai83) [inline docs/kubeedge.md]: MetaManager is the message processor ~~and~~ between edged and edgehub

Comment (anvithks) [reply docs/kubeedge.md]: This is fixed.

**Comment by sids-b** [comment]
@anvithks , can you please squash the commits ? Thanks

**Comment by anvithks** [comment]
> @anvithks , can you please squash the commits ? Thanks

This is done.

**Comment by anvithks** [comment]
@sids-b Please review and merge.

**Comment by anvithks** [comment]
There aren't any code changes so I guess you should be able to merge the PR. 
The tests seem to be stuck for a while now.

**Pull request merged**